woocommerce/woocommerce-gutenberg-products-block
# Add new links to experimental docs and update JS testing docs

As part of my porter duties I reviewed our current [documentation for feature flags and experimental interfaces](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/blocks/feature-flags-and-experimental-interfaces.md) and noticed there were some out of date/no longer used occurrences.

Specifically the largest change was removing links to out of date references in the `Product Price new controls ` section. The file `assets/js/atomic/blocks/product-elements/price/block.js` had been changed to use `ProductPrice` in #3065 so some of the `isFeaturePluginBuild()` calls were no longer used and this document was stale.

I added a line to the documentation about `__experimental_woocommerce_blocks_payment_gateway_features_list ` which was added in #3719 but not documented. 

This PR will bring that document in line with the current state of the project and provide a more accurate documentation of our use of experimental interfaces/feature flags.

I also made a change to our current [JS testing documentation](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/contributors/javascript-testing.md). Because it referenced Travis, and I also changed some version numbers in there to make it a little more up to date.

### How to test the changes in this Pull Request:

1. View the **new** [documentation for feature flags and experimental interfaces](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/d4f32877d8697752d621aff9b30915a3432d8c9a/docs/blocks/feature-flags-and-experimental-interfaces.md) 👀 and verify that it's correct.
2. View the **new** [documentation for JS testing](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/d4f32877d8697752d621aff9b30915a3432d8c9a/docs/contributors/javascript-testing.md) 👀 and verify it's correct.



### Changelog

> Improvements to documentation.
